### PR TITLE
Static linking of NetCDF to unit test targets

### PR DESCRIPTION
**Pull request by ahmad-el-sayed**
…ly to test targs

### CODE REVIEW

**Comment by ahmad-el-sayed** [comment]
@lucacarniato  This is going to fail because:

1. NetCDF module under linux has no cmake support 
2. NetCDF module under windows likely does not have the static libs, only the shared ones

**Comment by ahmad-el-sayed** [comment]
@lucacarniato  I stand corrected. Win build passed! 
I'll reinstall (possibly build from source) NetCDF on the build server tomorrow and make sure find_package can actually locate NetCDF.

**Comment by ahmad-el-sayed** [comment]
@lucacarniato  a couple of things:

1. Please do not merge until we figure out how to statically link netcdf on our machines. Otherwise local visual studio builds will fail.
2. Once item 1 is done, I need to update README.md (remove boost dep, add netcdf instructions)
3. In the mean time, I suggest that you use this branch to experiment with docker ([GRIDEDIT-468](https://issuetracker.deltares.nl/browse/GRIDEDIT-468) ) before we merge to master.

**Review by lucacarniato — APPROVED** [review]

**Pull request merged**
